hal_st/middlewares/cococo_modifications.md

```diff
@@ -0,0 +1,3 @@
+# CoCoCo Modifications
```

Comment:
There is no end of line.
Do we want "cococo" to be public?

Reply:
Removed CoCoCo from contents and filename. Added empty line.